### PR TITLE
practicals proposals

### PR DESCRIPTION
**Pull request by gilleslandais**
I propose a rought draft composed of practical sections (in markdown) (sorry for typo), that can be a basis for discussions (that can be linked in 'Issues') or to edit something more clean.

I don't know if the format (a directory containing proposals in dedicated markdown) suits you. If not, ignore my pull request.

regards!

### CODE REVIEW

**Comment by augustfly** [comment]
So basically you are proposing to work on sections in markdown instead of latex, and then once we are done we can merge those into the latex (as needed)?

**Comment by gilleslandais** [comment]
the method is may be original :)
It was mainly  to separate your edition to the proposals.
I chose markdown (for my proposals) more easy to edit (than to compile pdf).

File: proposals/ReadMe_gilles.md
Comment (augustfly) [inline proposals/ReadMe_gilles.md:1]: Please rename this "ReadMe.md"

Comment (gilleslandais) [reply proposals/ReadMe_gilles.md:1]: Ok, I renamed the files

**Pull request merged**